src/display/api.js

```diff
@@ -863,9 +874,9 @@ class PDFDocumentProxy {
  *                    just before viewport transform.
  * @property {Object} [imageLayer] - An object that has beginLayout,
  *                    endLayout and appendImage functions.
- * @property {Object} [canvasFactory] - The factory that will be used
+ * @property {Object} [canvasFactory] - The factory instance that will be used
```

Comment:
I'm *really* not happy about the discrepancy between `CMapReaderFactory` (on `getDocument`) and `canvasFactory`, since in the first case the user needs to pass a class and in the second one a class *instance*.
I've tried to make this at least *slightly* clearer by updating the JSDoc comment here, but it still doesn't seem great. Unfortunately it might simply be (years) too late to try and "fix" this, and furthermore I'm also not entirely sure how to improve it (given how `canvasFactory` is being used). Oh, well...